### PR TITLE
adjust package name for pkg/controller directory

### LINKED ISSUE #12741: controller package names don't match the folders
During the controller refactoring, the package names weren't made to match the folder names.  This is annoying an unexpected.

See https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/namespace/namespace_controller.go as an example.

### PR DESCRIPTION
**Pull request by zhengguoyong**
Fix #12741

### CODE REVIEW

**Comment by zhengguoyong** [comment]
@deads2k 
hi deaks2k, please make a review for this PR,
thanks.

**Comment by deads2k** [comment]
Looks like you accidentally make some files executable (100644 → 100755).  Can you change those back to `644`.

**Comment by zhengguoyong** [comment]
sorry about that
i changed it , please check it again , thanks.
@deads2k

**Comment by deads2k** [comment]
lgtm

File: contrib/mesos/pkg/scheduler/resource/resource.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ package resource
 import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
-	"k8s.io/kubernetes/pkg/controller/resourcequota"
+	resourcequotacontroller "k8s.io/kubernetes/pkg/controller/resourcequota"
```
Comment (liggitt) [inline contrib/mesos/pkg/scheduler/resource/resource.go]: it's unfortunate this package doesn't match its path

Comment (liggitt) [inline contrib/mesos/pkg/scheduler/resource/resource.go]: nm, I see that's cleaned up later

**Comment by liggitt** [comment]
LGTM

**Comment by liggitt** [comment]
@k8s-bot test this

**Comment by dchen1107** [comment]
GCE e2e test failure is due to known corrupted git workspace, reran the test now:

ok to test

**Pull request merged**